week-2/debug/0.js

```diff
@@ -3,5 +3,5 @@
 function multiply(a, b) {
   console.log(a * b);
 }
-
 console.log(`The result of multiplying 10 and 32 is ${multiply(10, 32)}`);
+//The console will take the function of multiply and evaluate the argument to 320 and it will return an undefined because the function was not called at the end.  
```

Comment:
Hey @sariat0001, here think more carefully about the difference between "return x" and "console.log(x)"

Reply:
And make sure you fix the bugin the code too!